docs/getting-started/brokers/sqs.rst

```diff
@@ -32,7 +32,17 @@ where the URL format is:
 
     sqs://aws_access_key_id:aws_secret_access_key@
 
-you must *remember to include the "@" at the end*.
+Please note that you must remember to include the ``@`` sign at the end and
+encode the password so it can be always parsed correctly. For example:
+
+.. code-block:: python
+
+    from kombu.utils.url import quote
+    
+    aws_access_key = quote("ABCDEFGHIJKLMNOPQRST")
+    aws_secret_key = quote("ZYXK7NiynGlTogH8Nj+P9nlE73sq3")
+    
+    broker_url = f"sqs://{aws_access_key}:{aws_secret_key}@"
```

Comment:
sadly it's python3 only for now and the next release of celery 4.x version will also support python 2.7. so kindly make it work in all supported version of python and we can switch to  f string from next major versions.

Reply:
I think this version (using `format`) is indeed better, as it covers all supported Python versions.

Reply:
OK, I updated the example - let me know if you'd like me to change anything else!